nmt_chainer/beam_search.py

```diff
@@ -105,6 +105,19 @@ def compute_next_lists(new_state_ensemble, new_scores, beam_width, eos_idx, curr
 #             if len(next_states_list) >= beam_width:
 #                 break
 
+    # Prune items that have a score worse than beam_size.
```

Comment:
Adjust comment: "Prune items that have a score worse than beam_size (*or beam_pruning_margin) below the best score"